security-analysis/security-analysis-api/src/main/java/com/powsybl/security/action/TapPositionAction.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.security.action;
+
+import java.util.Objects;
+
+/**
+ * An action changing the tap position of a phase-shifting transformer.
+ *
+ * @author Hadrien Godard <hadrien.godard@artelys.com>
+ */
+public class TapPositionAction extends AbstractAction {
+
+    public static final String NAME = "TAP_POSITION";
+
+    private final String transformerId;
+    private final int newTapPosition;
```

Comment:
I think that we need two modes:
- an absolute mode where you give the newTapPosition ;
- a relative mode where you give the tap variation.
For this, you need an `int` but `newTapPosition` is not the good term and a boolean (`delta`) that gives the mode. What do you think ?

Reply:
Also, as this action could be applied to two or three windings transformer, you should precise the `SIDE` that can be optional.

Reply:
Yes indeed.